common/lib/capa/capa/responsetypes.py

```diff
@@ -252,6 +256,59 @@ def evaluate_answers(self, student_answers, old_cmap):
         # log.debug('new_cmap = %s' % new_cmap)
         return new_cmap
 
+    def make_hint_div(self, hint_node, correct, label=None, hint_text=''):
+        """
+        Given the xml extended hint node and the boolean correctness, returns the extended hint div,
+        like this:
+          <div class="feedback_hint_incorrect">Incorrect: the answer is not 42</div>
+        Optionally, the hint label and the hint_text to use may be passed in,
+        overriding the text found in the hint node, which can be passed as None.
```

Comment:
I'm not sure what this means. The default for `hint_text` is `''`, not `None`, so are you saying that to get the text from the hint node I need to explicitly set `hint_text = None`?